docs/rules/no-confusing-arrow.md

```diff
@@ -65,6 +65,19 @@ var x = a => (1 ? 2 : 3);
 var x = (a) => (1 ? 2 : 3);
 ```
 
+`onlyOneSimpleParam` is a boolean setting that can be `true` or `false`(false):
+
+1. `true` warns if arrow functions with a single argument that is either an identifier or a primitive literal is used.
+2. `false` relaxes the rule.
+
+Examples of **incorrect** code for this rule with the `{"onlyOneSimpleParam": true}` option:
+
+```js
+/*eslint no-confusing-arrow: ["error", {"onlyOneSimpleParam": true}]*/
+/*eslint-env es6*/
+var x = (a) => (1 ? 2 : 3);
+```
+
```

Comment:
It's the opposite, `true` relaxes the rule. It would be better to add examples of correct code for this rule with the `{"onlyOneSimpleParam": true}` option, as that's the purpose of this option.

Reply:
To clarify, this is how the rule works without the option (i.e., with default `false`):

```js
/* eslint no-confusing-arrow: ["error"] */

a => 1 ? 2 : 3; // error
(a) => 1 ? 2 : 3;  // error

() => 1 ? 2 : 3;  // error
(a, b) => 1 ? 2 : 3;  // error
(a = b) => 1 ? 2 : 3;  // error
({ a }) => 1 ? 2 : 3;  // error
([a]) => 1 ? 2 : 3;  // error
(...a) => 1 ? 2 : 3;  // error
```

When the option is set to `true`, the rule checks only arrow functions with one simple param:

```js
/* eslint no-confusing-arrow: ["error", { onlyOneSimpleParam: true }] */

a => 1 ? 2 : 3; // error
(a) => 1 ? 2 : 3;  // error

() => 1 ? 2 : 3;  // ok
(a, b) => 1 ? 2 : 3;  // ok
(a = b) => 1 ? 2 : 3;  // ok
({ a }) => 1 ? 2 : 3;  // ok
([a]) => 1 ? 2 : 3;  // ok
(...a) => 1 ? 2 : 3;  // ok

```